frontend/www/js/omegaup/components/problem/Solution.vue

```diff
@@ -9,39 +18,38 @@
     <div v-else class="interstitial">
       <omegaup-markdown :markdown="statusMessage"></omegaup-markdown>
       <omegaup-markdown
-        v-show="allTokens !== null && availableTokens !== null"
+        v-show="allowedSolutionsToSee !== null"
         :markdown="
-          ui.formatString(T.solutionTokens, {
-            available: availableTokens,
-            total: allTokens,
-          })
+          ui.formatString(T.solutionViewsLeft, {
+            available: allowedSolutionsToSee,
+            total: 5,
+          }),
         "
       ></omegaup-markdown>
-      <div class="text-center">
+
+      <div class="text-center mt-5">
         <button
           v-if="status === 'unlocked'"
           class="btn btn-primary btn-md"
           @click="$emit('get-solution')"
         >
           {{ T.wordsSeeSolution }}
+          <font-awesome-icon :icon="['fas', 'unlock']" />
         </button>
         <button
-          v-else-if="
-            status === 'locked' &&
-            allTokens === null &&
-            availableTokens === null
-          "
-          class="btn btn-primary btn-md"
-          @click="$emit('get-tokens')"
+          v-else-if="status === 'locked' && allowedSolutionsToSee === null"
+          class="btn btn-secondary btn-md"
+          @click="$emit('get-allowed-solutions')"
         >
-          {{ T.solutionViewCurrentTokens }}
+          {{ T.solutionAvailableViews }}
         </button>
         <button
-          v-else-if="status === 'locked' && availableTokens &gt; 0"
+          v-else-if="status === 'locked' && allowedSolutionsToSee &gt; 0"
```

Comment:
Creo que el linter ya nos permite usar losoperadores correctamente en Vue, así que puedes intentar:
```suggestion
          v-else-if="status === 'locked' && allowedSolutionsToSee > 0"
```